src/test/java/StepDefinations/LoginFunctionality/LoginFunSeps.java

```diff
@@ -0,0 +1,74 @@
+package StepDefinations.LoginFunctionality;
```

Comment:
remove blank spaces
Remove commented code

Reply:
Done, I fixed this issue.

Reply:
resolved